code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -14,13 +14,17 @@
 /datum/species/jelly/on_species_loss(mob/living/carbon/C)
 	if(regenerate_limbs)
 		regenerate_limbs.Remove(C)
+	C.remove_language(/datum/language/slime)

```

Comment:
But what if they already had the slime language?

Look into using "shadow languages", like how the AI or Cyborg learn Common and Draconic and EAL by merit of their minds being in those bodies.

Reply:
oh, _that's_ what they're for!

Reply:
The name is because the mind datum (over)shadows the body's language datum? I don't know, that's just what they're for.